project/src/components/reviews-form/reviews-form.tsx

```diff
@@ -10,22 +15,36 @@ const ratings = [
 ];
 
 function ReviewsForm(): JSX.Element {
-  const [state, setState] = useState({rating: 0, review: ''});
+  const {id} = useParams() as {id: string};
```

Comment:
```suggestion
  const {id} = useParams<{id: string}>();
```
а лучше вынеси в отельный тип